netci.groovy

```diff
@@ -433,7 +437,7 @@ def static addNonPRTriggers(def job, def branch, def isPR, def architecture, def
                     }
                 }
                 // For x86, only add per-commit jobs for Windows
-                else if (architecture == 'x86') {
```

Comment:
Lets see if we can live without them for now.  R2R is important for perf testing, and perf is the main reason for legacy_backend, so we may have to reevaluate.